cmd/can/main.go

```diff
@@ -21,15 +22,19 @@ func main() {
 	if config.Debug {
 		fmt.Printf("Reading API specification from \"%s\"\n", cfg.GetOpenAPIFilepath())
 	}
+	// TODO: What if JSON?
+	// TODO: MustLoadOpenApiFile
```

Comment:
How do you feel about contributing a style guide item to the repo that explains your use of Musty functions? I agree that they should be used here, but I want to document the trade offs and caveats. Musty functions are rigid. They enforce a behaviour on the consumer. Do we provide non musty versions for testing? Do we write a non musty and then a musty version of each function?